### PR TITLE
Keep acronyms upper case

### PR DESCRIPTION
**Pull request by sachaarbonel**

### CODE REVIEW

File: packages/stream_feed/lib/src/core/api/images_api.dart [hunk 1]
```diff
@@ -4,9 +4,9 @@ import 'package:stream_feed/src/core/http/token.dart';
 import 'package:stream_feed/src/core/util/routes.dart';
 
 /// The http layer api for CRUD operations on Images
-class ImagesApi {
+class ImagesAPI {
   /// [ImagesApi] constructor
```
Comment (xsahil03x) [inline packages/stream_feed/lib/src/core/api/images_api.dart:8]: ImagesAPI

File: packages/stream_feed/test/mock.dart [hunk 2]
```diff
@@ -27,21 +27,21 @@ class MockDio extends Mock implements Dio {
...
+class MockFilesAPI extends Mock implements FilesAPI {}
 
-class MockReactionsApi extends Mock implements ReactionsApi {}
+class MockReactionsAPI extends Mock implements ReactionsAPI {}
 
-class MockImagesApi extends Mock implements ImagesApi {}
+class MockImagesAPI extends Mock implements ImagesAPI {}
 
-class MockUsersApi extends Mock implements UsersApi {}
+class MockUsersApi extends Mock implements UsersAPI {}
```
Comment (xsahil03x) [inline packages/stream_feed/test/mock.dart:40]: MockUsersAPI

File: packages/stream_feed/lib/src/core/api/images_api.dart (hunk 1 above)
Comment (xsahil03x) [reply packages/stream_feed/lib/src/core/api/images_api.dart:8]:
```suggestion
  /// [ImagesAPI] constructor
```

File: packages/stream_feed/test/mock.dart (hunk 2 above)
Comment (xsahil03x) [reply packages/stream_feed/test/mock.dart:40]:
```suggestion
class MockUsersAPI extends Mock implements UsersAPI {}
```

Might also need to refactor wherever it is used.

File: packages/stream_feed/lib/src/core/api/images_api.dart (hunk 1 above)
Comment (xsahil03x) [inline packages/stream_feed/lib/src/core/api/images_api.dart:8]:
```suggestion
  /// [ImagesAPI] constructor
```

File: packages/stream_feed/test/mock.dart (hunk 2 above)
Comment (xsahil03x) [inline packages/stream_feed/test/mock.dart:40]:
```suggestion
class MockUsersAPI extends Mock implements UsersAPI {}
```

Might also need to refactor wherever used.

File: packages/stream_feed/lib/src/core/api/images_api.dart (hunk 1 above)
Comment (sachaarbonel) [reply packages/stream_feed/lib/src/core/api/images_api.dart:8]: good catch

**Review by xsahil03x — APPROVED** [review]
LGTM

**Pull request merged**